scripts/find-slow-tests.js

```
@@ -0,0 +1,158 @@
+#!/usr/bin/env node
+/* eslint-env node */
+/* global require, process, module */
+/**
+ * Script to identify slow-running tests
+ * Usage: node scripts/find-slow-tests.js
+ */
+
+const { spawn } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)